### PR TITLE
configs: add minimal configs for apu{3,4,5} and remove redundant ones

### PR DESCRIPTION
**Pull request by pietrushnic**
Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>

### CODE REVIEW

File: configs/config.pcengines_apu3 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CONFIG_VENDOR_PCENGINES=y
+CONFIG_BOARD_PCENGINES_APU3=y
+CONFIG_NO_GFX_INIT=y
+CONFIG_PXE=y
+CONFIG_BUILD_IPXE=y
+CONFIG_PXE_ROM_ID="8086,157b"
```
Comment (miczyg1) [inline configs/config.pcengines_apu3]: 8086,1539 for apu3/4/5

File: configs/config.pcengines_apu4 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CONFIG_VENDOR_PCENGINES=y
+CONFIG_BOARD_PCENGINES_APU4=y
+CONFIG_NO_GFX_INIT=y
+CONFIG_PXE=y
+CONFIG_BUILD_IPXE=y
+CONFIG_PXE_ROM_ID="8086,157b"
```
Comment (miczyg1) [inline configs/config.pcengines_apu4]: as above

File: configs/config.pcengines_apu5 [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CONFIG_VENDOR_PCENGINES=y
+CONFIG_BOARD_PCENGINES_APU5=y
+CONFIG_NO_GFX_INIT=y
+CONFIG_PXE=y
+CONFIG_BUILD_IPXE=y
+CONFIG_PXE_ROM_ID="8086,157b"
```
Comment (miczyg1) [inline configs/config.pcengines_apu5]: as above

File: configs/config.pcengines_apu2 [hunk 4]
```diff
@@ -9,5 +9,5 @@ CONFIG_BUILD_IPXE=y
 CONFIG_PXE_ROM_ID="8086,157b"
 CONFIG_MEMTEST_SECONDARY_PAYLOAD=y
 CONFIG_CPU_MICROCODE_CBFS_NONE=y
-CONFIG_PXE_CUSTOM_GENERAL_H="general.h"
```
Comment (miczyg1) [inline configs/config.pcengines_apu2]: No more embedding scripts into iPXE?

Comment (pietrushnic) [reply configs/config.pcengines_apu2]: @miczyg1 yes, please note that I will set this as default in config options, this limit number of custom changes that mainline have to accept

**Pull request merged**